core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.ts

```diff
@@ -267,7 +267,10 @@ angular.module('oppia').directive('skillsList', [
                 $timeout(function() {
                   TopicsAndSkillsDashboardBackendApiService.
                     onTopicsAndSkillsDashboardReinitialized.emit();
-                }, 100);
+                }, 100).then(function() {
+                  var successToast = 'Merged Skills.';
+                  AlertsService.addSuccessMessage(successToast, 1000);
+                });
```

Comment:
Hm, can you please explain why we show the alert text in the `then` part of $timeout? Can we call it inside the $timeout (right after emit() is called)?

Reply:
oh my bad, I wanted to put it in the ```then``` part of the previous ```then```. We can also put it just after emit() too.

Reply:
OK can you also please update the "Proof that changes are correct" section of this PR? Thanks!

Reply:
Done